Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -137,6 +140,25 @@ public final class EditorialProjectsViewController: UIViewController {
           }
       }
 
+    self.viewModel.outputs.closeButtonImageTintColor
+      .observeForUI()
+      .observeValues { [weak self] color in
+        _ = self?.closeButton
+          ?|> UIButton.lens.image(for: .normal) .~ image(named: "icon--cross", tintColor: color)
+      }
+
+    self.viewModel.outputs.setNeedsStatusBarAppearanceUpdate
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.setNeedsStatusBarAppearanceUpdate()
+      }
+
+    self.viewModel.outputs.applyViewTransformsWithY
+      .observeForControllerAction()
```

Comment:
Snapshot tests simply wouldn't pass without this. This VC is doing something really strange with the refreshControl and it's breaking a number of things, I think.